code/modules/psionics/faculties/coercion.dm

```diff
@@ -248,18 +248,22 @@
 			to_chat(user, SPAN_CULT("Not even a psion of your level can speak to the dead."))
 			return
 
-		if (target.isSynthetic())
-			to_chat(user, SPAN_WARNING("This can only be used on living organisms."))
-			return
-
+		var/obj/item/organ/internal/augment/nlom/nlomaug
```

Comment:
Change the var name to reflect the function of the implant, not how its called in lore.